contents/docs/data/event-spec/ecommerce-events.mdx

```diff
@@ -0,0 +1,1212 @@
+---
+title: Ecommerce events specification
+sidebar: Docs
+showTitle: true
+---
+
+PostHog's ecommerce spec helps define the journey for a customer as they browse your store, click on promotions, view products, add those products to a cart, and complete a purchase.
+
```

Comment:
```suggestion

This spec serves as the foundation for our CDP destination templates. We recommend implementing this spec to avoid having to manually map events and properties to our CDP destinations.

```